src/app/app.js

```diff
@@ -1,4 +1,4 @@
-import '@webcomponents/webcomponentsjs/bundles/webcomponents-ce';
+import 'document-register-element/build/document-register-element'; // ES2015
```

Comment:
Well that is broken...
https://github.com/WebReflection/document-register-element/issues/155